uiowa/uiowa
# Restrict more blocks

Heavy-handed approach for v3 default until we can wrangle all of the formatters each one of these blocks allows and causes errors with.

Resolves https://github.com/uiowa/uiowa/issues/1920 by removing blocks that can be placed and restricting block configuration for the remaining ones.
Resolves https://github.com/uiowa/uiowa/issues/2002 and resolves https://github.com/uiowa/uiowa/issues/1999 by removing blocks from being placed.


## To Test

- As a webmaster/editor on a v3 default site, you will find that the "More..." blocks are much fewer than before. Only body, main navigation and webform remain. It shouldn't affect existing blocks except that if you remove them, you may not be able to replace it again.
- Of the blocks that remain, make sure you can't cause errors similar to #2002 and #1999.
- The main navigation block should allow you to pick style orientation, but not set other config. It should set defaults for the active trail menu pattern (start at second level, follow active trail)

Color issues will be addressed in the other PR #2069
